workspaces/optic-engine-native/package.json

```diff
@@ -4,9 +4,9 @@
   "scripts": {
     "postinstall": "node scripts/install",
     "preuninstall": "node scripts/uninstall",
-    "ws:build": "scripts/build-binaries.sh",
+    "ws:build": "node scripts/uninstall && cargo build",
```

Comment:
These should be equivalent commands to the shell scripts - so there shouldn't be any impact outside of the windows regression run now being able to run this